src/styles/StyleUtils.ts

```diff
@@ -58,13 +59,19 @@ type ModalPaddingStylesArgs = {
     insets: EdgeInsets;
 };
 
-type AvatarBorderStyleArgs = {
+type AvatarBorderStyleParams = {
     isHovered: boolean;
     isPressed: boolean;
     isInReportAction: boolean;
     shouldUseCardBackground: boolean;
 };
 
+type GetBaseAutoCompleteSuggestionContainerStyleParams = {
```

Comment:
why do we need to add this type?

Reply:
@hayata-suenaga not sure what do you mean?

Reply:
ah never mind i got it